### PR TITLE
Consolidate and expand statistics page

### LINKED ISSUE #1107: Make it easy to identify new potential volunteer administrators
I've noticed a few enthusiastic users on our Alaveteli site helpfully making annotations on new requests. But now I can't remember who those users are and as an administrator I have no way of tracking them down and asking if they want to help being a site administrator.

I think the easiest way to solve this is to have a section in the admin backend that allows me to see users with the most annotations. Maybe just a sort field on the user page?

### LINKED ISSUE #3545: Show number of hidden requests over time
In thinking about #3515 we [realised](https://github.com/openaustralia/righttoknow/issues/584#issuecomment-251577658) we need a way to see if we're having an impact on the number of requests for personal information. One way of doing this is to look at the volume of requests that are hidden over time. Of course this is imperfect as there are other reasons you might hide a request but it seems like a good start.

We're thinking of adding something like these charts on [PlanningAlerts authority pages](https://www.planningalerts.org.au/authorities/marrickville) to the new statistics page proposed in #3544:

selection_001

This relates to #2658 because it's a vastly simpler first step in that direction.

### PR DESCRIPTION
**Pull request by henare**
This pull request moves the public body statistics page onto a general statistics page and adds two new sections:
1. The Humans of Right To Know user statistics added to Right To Know as part of openaustralia/righttoknow#645
2. A chart that tracks hidden requests over time, see #3545

For more information about the reasoning of these changes, please see the issues linked above.

As part of making these changes we've done the following supporting work:
- 3b43c3142ca3096206bae82b90dd924343df135c redirect requests to old public body statistics page to new general statistics page
- Extract a Statistics [model](https://github.com/mysociety/alaveteli/compare/develop...openaustralia:3545-hidden-requests-chart#diff-1673fd6e2a9e982fd66034bf3f9b006a), [controller](https://github.com/mysociety/alaveteli/compare/develop...openaustralia:3545-hidden-requests-chart#diff-994cde5bf25cf299666d1bad6612b58d) and assets files to tidy things up
- All the statistics are available nested [under the same API](https://github.com/mysociety/alaveteli/compare/develop...openaustralia:3545-hidden-requests-chart#diff-994cde5bf25cf299666d1bad6612b58dR16)
- db72f3119d66e10a57688cd0993bee74f99b286a When an administrator uses the backend to only update the prominence of a request to hide it we now treat this as a hide event, not a regular edit event. As mentioned on #3543, we've been using this method exclusively to hide requests - this change (and the following migration) makes them correctly appear as hide events
- 03a559f82f456542e0aa370eafdc46abb2ce30cc Add a migration to update event types of existing hide events
- fa148ec621c561cfb06d9591f55d27d99fa07170 Small bugfix to allow params yaml of an InfoRequestEvent to be blank instead of raising an exception

Fixes #1107.
Fixes #3545.

alaveteli_statistics

### CODE REVIEW

**Comment by henare** [comment]
Apologies for the epic pull request :mount_fuji: but we've decided to update this to also incorporate our recent work on #3545 as it actually seems like it will be easier to review it all in one go :shipit: 

We've tried to describe all the major changes we've done along the way but if you have any questions then please ping @equivalentideas or I.

File: doc/CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Highlighted Features
 
+* New statistics page that includes user stats to show top requesters and
+  annotators, and hidden  requests. Includes a new event type of "hide" to
+  make tracking and reporting on hidden requests much simpler. Need to run
+  `rake temp:update_hide_event_type` to set up the data for this feature
+  (Henare Degan)
```
Comment (henare) [inline doc/CHANGES.md]: Can we please change this to `(Henare Degan, Luke Bacon)`. Although the commits are attributed to me we paired on this entire feature.

**Comment by henare** [comment]
Amazing, thanks @lizconlan! :star2: I've just left a little comment about attribution that I hope can be changed before this is merged :smile:

File: lib/tasks/temp.rake [hunk 2]
```diff
@@ -1,6 +1,15 @@
 # -*- coding: utf-8 -*-
 namespace :temp do
 
+  desc 'Update EventType when only editing prominence to hide'
```
Comment (equivalentideas) [inline lib/tasks/temp.rake]: @lizconlan I'm interested in why you've moved this to a rake task? Is it a convention we should be following for future contributions, or something specific to what this does?

File: doc/CHANGES.md (hunk 1 above)
Comment (lizconlan) [inline doc/CHANGES.md]: Sure, I'll fix that right up - sorry @equivalentideas!

**Pull request merged**